priv/templates/phoenix.gen.html/haml/edit.html.haml

```diff
@@ -0,0 +1,7 @@
+%h2
+  Edit <%= template_singular %>
```

Comment:
can you just do `#{template_singular}` here instead of the eex?.  Not sure about that in this haml implementation...